google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/AddCustomerMatchUserList.java

```diff
@@ -256,6 +256,8 @@ private void addUsersToCustomerMatchUserList(
       System.out.println("Waiting until operation completes.");
 
       // The polling future implements a default back-off policy for retrying.
+      // Note: Retrieving the results of the asynchronous request is for demonstrative purposes only
```

Comment:
Can we give guidance for how to restart polling asynchronously? It seems like this could be a source of confusion...

Reply:
I experimented with getting the LRO handle again when I worked on the Stores Sales upload example, and found that process confusing as well. This is why that example just prints out a GAQL query the user can use to come back and check on the results of the LRO later:

https://github.com/googleads/google-ads-java/blob/master/google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadStoreSalesTransactions.java#L494

WDYT of going with that approach here?

Reply:
SGTM

Reply:
SGTM as well. I've also added an option to print the results if the status is SUCCESS.